digitkraft-backend/pom.xml

```diff
@@ -62,6 +63,16 @@
 			<groupId>org.flywaydb</groupId>
 			<artifactId>flyway-core</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>javax.validation</groupId>
+			<artifactId>validation-api</artifactId>
+			<version>${javax.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
```

Comment:
I added missing Maven dependencies, but next time remember to add them to the repository.